server/views/pages/addPatient/addPatientSelectHospital.njk

```diff
@@ -58,14 +58,14 @@
         <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
 
         {{ govukSelect({
-          id: "hospital",
+          id: "agency",
```

Comment:
note that only changing the `id` here, the `name` is unchanged so will be submitted as normal